app/assets/javascripts/comments.js.coffee

```diff
@@ -0,0 +1,17 @@
+@CommentPoller =
+	poll: ->
+		setTimeout @request, 30000
```

Comment:
How about putting that number in some variable for configuration? I don't think it should be hard coded.